themes/reactos/layouts/partials/recent_posts.html

```diff
@@ -46,6 +46,8 @@ <h3>Live Activity</h3>
 			</div>
 
 			<ul id="jirafeed"></ul>
+			
+			<a href="https://jira.reactos.org/secure/Dashboard.jspa#Activity-Stream/10001">More...</a>
```

Comment:
```suggestion
			<a href="https://jira.reactos.org">More...</a>
```

Reply:
Thanks @learn-more. If a short URL is better, I'm OK with this change.
Btw, I simply copied that URL from old website's link (which I backuped locally before site update), and thought that it should be the same as earlier. From start, I already added the short link, but then decided to use the long one instead.